packs/feats/megavolt.json

```diff
@@ -24,7 +24,31 @@
             "remaster": false,
             "title": "Pathfinder Guns & Gears"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "label": "PF2E.TraitUnstable",
+                "mergeable": true,
```

Comment:
?

Reply:
Same as the other ones, this ones has an Unstable function that should add the Unstable trait when used.

Reply:
Removed all instances of mergeable: true from the RE.